### ISSUE TITLE
Problems installing lightbi

### ISSUE THREAD

**Issue by satyagrahi** [binokochumolvarghese/lightbi-hugo#56]
Thanks for creating the theme, i like it!

However, trying to get it running has been difficult:

I copied your example site/content into /content, but need to copy more to get menus, etc.
How/where do i put the _index.md? Or do i have to let Hugo generate it only?
How do i get search working?
Looked at i18n but was not clear on next steps.

Thanks. Can use hugo discourse or any other means of communication.
rajesh@peerwater.org

**Comment by binokochumolvarghese** [comment]
Hi,

Thanks for the kind words and sorry for the troubles.

Can you check the full working demo [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo) or my [blog](https://binovarghese.com/blog/2021/build-your-first-hugo-blog/) which showing the full steps to install this theme.

**Comment by satyagrahi** [comment]
Hi,
Thank you for responding.
When i downloaded your example website last week, i got the latest version.
My 3 questions above still are not answered in your blog.
If possible, please answer, else i will keep puttering around :)
Thanks in advance

**Comment by binokochumolvarghese** [comment]
Hi,
You have to copy the `hugo.toml` in the content to the root folder for menu and other site parameters. Check the latest version, earlier the name was `config.toml`. I have mentioned the same in the blog under **Modify Configuration** section.

`_index.md` is required for every section in you blog like posts, notes, and collections. It is also available in the content folder.

Check [here](https://github.com/binokochumolvarghese/lightbi-hugo/wiki/General-Features#multilingual) for multilanguages.

Please let me know the above helped or not.

**Comment by satyagrahi** [comment]
Hi,
I tried on a windows machine from the very first instruction. Got everything setup, including copying the hugo.toml file.
I get this error, which i do not understand:

`> hugo server
Watching for changes in C:\Users\rajesh\Hugo\lightbi\{archetypes,assets,content,data,i18n,layouts,static,themes}
Watching for config changes in C:\Users\rajesh\Hugo\lightbi\hugo.toml
Start building sites …
hugo v0.123.1-3f8434a62fc91a50f4e5bd43a4435d2d301844ff+extended windows/amd64 BuildDate=2024-02-21T08:17:45Z VendorInfo=gohugoio

ERROR The "tweet" shortcode requires two named parameters: user and id. See "C:\Users\rajesh\Hugo\lightbi\content\post\2020\jan\rich-content.md:24:1"
WARN  The author key in site configuration is deprecated. Use params.author.email instead.
WARN  The author key in site configuration is deprecated. Use [params.author.name](http://params.author.name/) instead.
Built in 1139 ms
Error: error building site: logged 1 error(s)`

**Comment by satyagrahi** [comment]
Hi,
Got everything work, including search, and its great. Now i can work on my content.
Just a couple of issues:
- your new theme download still has the tweet author/param problem. I had to download the rich-content.md file separately.
- the new theme example has an extra 'content' directory with 'layouts' in it. I had to move the layouts directory one level up and move the files from content/content up one level.
Thank you, hope to release my site in a week.
